cogs/social.py

```diff
@@ -169,12 +170,17 @@ async def lyrics(self, ctx, *, title):
         async with aiohttp.ClientSession() as cs:
             async with cs.get(f"https://some-random-api.ml/lyrics?title={title}") as r:
                 js = await r.json()
-
-                e = discord.Embed(color=discord.Color.random(), title=js['title'])
-                e.set_author(name=js['author'], url=js['links']['genius'])
-                e.description = js['lyrics']
-
-                await ctx.send(embed=e)
+                
+                lyricsArray = js['lyrics'].split("\n")
```

Comment:
This splits it for every line. If you wanted to split it between every verse you would have to do `\n\n`.
However, this is not what Flitz intended the code to do. They wanted it to split every 2000 characters (not between lines). I'll fix this issue for you because Flitz said I should fix it.